FinniversKit/Sources/Components/HelthjemView/HelthjemView.swift

```diff
@@ -0,0 +1,239 @@
+import UIKit
+
+public protocol HelthjemViewDelegate: AnyObject {
+    func helthjemViewDidSelectPrimaryButton(_ view: HelthjemView)
+    func helthjemViewDidSelectSecondaryButton(_ view: HelthjemView)
+}
+
+public final class HelthjemView: UIView {
+    public weak var delegate: HelthjemViewDelegate?
+
+    // MARK: - Subviews
+
+    private lazy var helthjemImage: UIImage = {
+        let lightImage = UIImage.init(named: .shipWithHelthjem)
+        let darkImage = UIImage.init(named: .shipWithHelthjemDarkmode)
+
+        if #available(iOS 13.0, *) {
+            switch Config.userInterfaceStyleSupport {
```

Comment:
I think this can be simplified to just check `traitCollection.userInterfaceStyle` , if it's `.dark` or `.light`. 
And, if you want the image to update to the correct color if the user switched between light and dark mode while being on the object page, you have to `override traitCollectionDidChange` to update the image (but it might be acceptable to keep the image as is, since it's just an illustration, it's not super bad if it doesn't update 😛 )